admin/src/collections/index.ts

```diff
@@ -0,0 +1,17 @@
+import { EntityReference } from '@camberi/firecms';
+
+export interface BuildCollectionProps {
+	isGlobalAdmin: boolean;
+	organisations?: EntityReference[];
+}
+
+export { adminsCollection } from './Admins';
```

Comment:
Out of curiosity: what is the advantage of exporting the collections again here (and for the interfaces) instead of just importing them directly when using them? To have an easy overview of the available ones?

Reply:
Yes, I think that was my intention :) We can also use direct imports, I have no strong preference...